arch/arm64/configs/deepin_arm64_desktop_defconfig

```diff
@@ -757,10 +757,13 @@ CONFIG_PCI_HOST_THUNDER_PEM=y
 CONFIG_PCI_HOST_THUNDER_ECAM=y
```

Comment:
Can we split defconfig changes into a separate pull request? This would make it easier for me to track on the rolling end.

Reply:
of course

Reply:
done.